pkg/awss3/storage.go

```diff
@@ -11,19 +11,17 @@ import (
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/s3"
 	"github.com/aws/aws-sdk-go/service/s3/s3manager"
+	"github.com/hypnoglow/helm-s3/pkg/awsutil"
```

Comment:
Just a trifle, but I prefer to put project/company imports separately from other 3rd party imports. **goimports** tool [supports this](https://github.com/golang/go/issues/12420).

Reply:
done!